src/Common/tests/System/AssertExtensions.cs

```diff
@@ -24,6 +24,12 @@ public static void Throws<T>(string netCoreParamName, string netFxParamName, Act
         {
             T exception = Assert.Throws<T>(action);
 
+            if (netFxParamName == null)
+            {
+                // Param name varies between NETFX versions -- skip checking it
+                return;
+            }
```

Comment:
I don't understand this change.  Doesn't this mean we stop validating the netCoreParamName if netFxParamName is null?